vmslave-config/pipeline/vmslave_config.groovy

```diff
@@ -0,0 +1,29 @@
+def checkout(String url, String branch, String targetDir){
+    checkout(
+    [$class: 'GitSCM', branches: [[name: branch]],
+    extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: targetDir]],
+    userRemoteConfigs: [[url: url]]])
+}
+
+node('vmslave-config') {
```

Comment:
This slave need to be set:
1. installed ansible 
2. a ansible hosts file(details in readme of this pr)
so in our ci system, I setup slaves with label vmslave-config.